### PR TITLE
fix: <Select/> width prop only applies to the input inside

### PR DESCRIPTION
**Pull request by balzss**
INSTUI-3820

test plan:
- add `width="8rem"` to a `Select` or `SimpleSelect` component
- click to the right of the component -> the dropdown shouldn't open

### CODE REVIEW

File: packages/ui-select/src/Select/index.tsx [hunk 1]
```diff
@@ -673,6 +673,13 @@ class Select extends Component<SelectProps> {
     const highlightedOptionId = this.highlightedOptionId
     const selectedOptionId = this.selectedOptionId
 
+    // props.width should apply to the whole component, not just the input in it
+    const extendedStyle = {
+      ...(styles?.select as object),
+      width: this.props.width || '100%',
+      display: 'inline-block'
+    }
```
Comment (matyasf) [inline packages/ui-select/src/Select/index.tsx:681]: CSS should be in `styles.ts`. This should be in the `select` object there.

**Review by matyasf — CHANGES_REQUESTED** [review]
see my comments

File: packages/ui-select/src/Select/index.tsx (hunk 1 above)
Comment (balzss) [reply packages/ui-select/src/Select/index.tsx]: good point, moved it there

**Review by matyasf — APPROVED** [review]

**Review by matyasf — COMMENTED** [review]
Can you please check Chromatic? It shows visual changes, this should not happen, this'd be a breaking change.

**Review by matyasf — APPROVED** [review]

**Review by joyenjoyer — APPROVED** [review]

**Pull request merged**